docs/examples/minhash-dedupe.md

```diff
@@ -111,12 +108,11 @@ df_warc.select("WARC-Identified-Payload-Type").distinct().show()
 
 ## Preprocessing
 
-Since we are primarily concerned with text, we will focus on `text/html` payloads, extracting text content from html body and normalizing the text itself.
+Since we are primarily concerned with text, we will focus on `text/html` payloads, extracting text content from html body and normalizing the text itself. Common crawl also comes with text only .wet files that come preprocessed, but here we choose to handle each html block explicitly to ensure consistent comparisons across pages.
```

Comment:
```suggestion
Since we are primarily concerned with text, we will focus on `text/html` payloads, extracting text content from html body and normalizing the text itself. Common Crawl also comes with text only .wet files that come preprocessed, but here we choose to handle each html block explicitly to ensure consistent comparisons across pages.
```